tests/main.test.ts

```diff
@@ -16,6 +16,12 @@ beforeEach(async () => {
   process.env.COREPACK_DEFAULT_TO_LATEST = `0`;
 });
 
+afterEach(async () => {
+  const home = npath.toPortablePath(process.env.COREPACK_HOME!);
+  await xfs.removePromise(home, {recursive: true});
+});
+
```

Comment:
Let's return a teardown function instead, should be cleaner. Could you also add this to the other tests please?
```suggestion
  return () => xfs.removePromise(home, {recursive: true});
});
```

Reply:
Yes, that’s definitely cleaner. Fixed in https://github.com/nodejs/corepack/pull/662/commits/da69ce3d4c579290a8e262350ed955858e586183